src/components/whiteboards/toolbar/Toolbar.vue

```diff
@@ -64,6 +64,7 @@ export default {
     unlock() {
       this.setDisableAll(false)
       this.selected = undefined
+      console.log(JSON.stringify(this.$canvas))
```

Comment:
@johncrossman is this log statement left here intentionally?

Reply:
@lyttam - No, I'll remove.